frontend/src/app/reservation-check/reservation-check.component.ts

```
@@ -0,0 +1,57 @@
+    private messageService: MessageService
+  ) { }
+
+  ngOnInit() {
+    // Fetch the language code from the URL
+    this.route.params.subscribe(params => {
+      this.languageCode = params['code'] || 'en'; // Default to 'en' if missing
+
+    });
+  }
+  goToCheckIn() {
+    var _ = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)